packages/web/src/executors/file-server/file-server.impl.ts

```diff
@@ -1,12 +1,12 @@
-import { exec, execSync } from 'child_process';
 import { ExecutorContext, joinPathFragments } from '@nrwl/devkit';
-import ignore from 'ignore';
+import { workspaceLayout } from '@nrwl/workspace/src/core/file-utils';
+import { exec, execSync } from 'child_process';
+import { watch } from 'chokidar';
 import { readFileSync } from 'fs';
+import ignore from 'ignore';
 import { Schema } from './schema';
```

Comment:
My editor organized the imports. I like it, but if you want me to reset this, let me know.